packages/slate-editor/src/extensions/snippet/lib/useFloatingSnippetInput.ts

```diff
@@ -42,8 +42,10 @@ export function useFloatingSnippetInput(editor: Editor): [State, Actions] {
 
         try {
             EditorCommands.insertNodes(editor, node.children);
+            editor.blink(node.children.at(0), node.children.at(-1));
             savedSelection.restore(editor, { focus: true });
         } catch (error) {
+            console.error(error);
```

Comment:
Should it be there for the production code? (just checking if it's not something you've forgot to remove)